subprojects/core/src/main/groovy/org/gradle/api/internal/artifacts/dsl/DefaultRepositoryHandler.java

```diff
@@ -37,6 +37,9 @@
     public static final String DEFAULT_BINTRAY_JCENTER_REPO_NAME = "BintrayJCenter";
     public static final String BINTRAY_JCENTER_URL = "https://jcenter.bintray.com/";
 
+    public static final String DEFAULT_JITPACK_REPO_NAME = "JitPack";
+    public static final String JITPACK_URL = "https://jitpack.io"
```

Comment:
What a a great idea! Thank you @championswimmer! Looks like there's a semicolon missing at the end of this line